### PR TITLE
Update prepare-release.yml

### PR DESCRIPTION
**Pull request by georgettica**
this should bump versions and make the code be more compliant

the issue we had before in prev PR seems to be because of environments we are not setting

### CODE REVIEW

**Comment by georgettica** [comment]
might solve #423

**Comment by georgettica** [comment]
@mikepea are you still an active maintainer in this repo?
if so I would love to get the +1 from you on this

tested partially on my machine and it looked to do what I wanted (just saw the binary is in the same name and location.. didn't check api changes yet

**Comment by ldelossa** [comment]
This looks good to me.

**Review by ldelossa — APPROVED** [review]

**Pull request merged**
